sk0-back/src/users/dtos/createUser.dto.ts

```diff
@@ -0,0 +1,20 @@
+import { IsEmail, IsNotEmpty, Matches } from 'class-validator';
+
+export class CreateUserDto {
+  id: number;
+
+  @IsNotEmpty()
+  username: string;
+
+  @IsNotEmpty()
+  @IsEmail()
+  email: string;
+
+  @IsNotEmpty()
+  // @MinLength(6)
+  @Matches(/^(?=.*[a-z])(?=.*[A-Z])(?=.*\d).{6,}$/, {
```

Comment:
Ezt majd látod nálam:

```suggestion
  @IsStrongPassword(passwordRules) {
```

Reply:
módosítva